db/migrate/20160808150745_add_metrics_status_to_ext_management_system.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetricsStatusToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :last_metrics_error, :text
+    add_column :ext_management_systems, :last_metrics_update_date, :timestamp
+    add_column :ext_management_systems, :last_metrics_success_date, :timestamp
```

Comment:
I think `last_metrics_status`, since it holds ["Valid"](https://github.com/ManageIQ/manageiq/pull/9785/files#diff-189fb4a1163fabfb715cd145f5da6edaR68) as well as failure states like ["Unavailable"](https://github.com/ManageIQ/manageiq/pull/9785/files#diff-189fb4a1163fabfb715cd145f5da6edaR62).